src/_locales/en/messages.json

```
@@ -0,0 +1,109 @@
+  },
+  "notInstalledErrorAction": {
+    "message": "Download Firefox"
+  },
+  "openWelcomePage": {
+    "message": "Extension Settings"
+  },
+  "welcomePageTitleFirefox": {
+    "message": "Launch in Another Browser"
+  },
+  "welcomePageSubtitleFirefox": {
+    "message": "Switch between browsers in one click."
+  },
+  "welcomePageDescriptionFirefox": {
+    "message": "Sometimes, you just want to use a different browser.\nThis extension helps you easily open sites in other browsers."
+  },
+  "welcomePageTitleChromium": {
+    "message": "Launch in Firefox"
+  },
+  "welcomePageSubtitleChromium": {
+    "message": "Switch from your current browser to Firefox in one click."
+  },
+  "welcomePageDescriptionChromium": {
+    "message": "Sometimes, you just want a more private browser. This extension helps you easily open sites in Firefox."
+  },
+  "welcomePageBrowserSelector": {
+    "message": "Select other browser"
+  },
+  "welcomePageAlwaysPrivateCheckbox": {
+    "message": "Always open sites in a Firefox Private window when switching"
+  },
+  "welcomePageTelemetryCheckbox": {
+    "message": "Allow Mozilla to collect information about how you use this extension so we can make it better. {LinkStart}Privacy Notice{LinkEnd}"
+  },
+  "welcomePageNoShortcutsFirefox": {
+    "message": "You don't have any shortcuts set up for this extension. Create them by going to {LinkStart}extension settings.{LinkEnd}"
+  },
+  "welcomePageNoShortcutsChromium": {
+    "message": "You don't have any shortcuts set up for this extension. Create them by going to {LinkStart}extension settings.{LinkEnd}"
+  },
+  "welcomePageNoShortcutTo": {
+    "message": "You don't have a shortcut set up for launching your current page in $browser$",
+    "placeholders": { "browser": { "content": "$1" } }
+  },
+  "welcomePageYesShortcutTo": {
+    "message": "$hotkey$ launches your current page in $browser$",
+    "placeholders": {
+      "hotkey": { "content": "$1" },
+      "browser": { "content": "$2" }
+    }
+  },
+  "welcomePageShortcutTitle": {
+    "message": "Shortcuts"
+  },
+  "welcomePageManageShortcutsFirefox": {
+    "message": "Find shortcuts for this extension by going to {LinkStart}extension settings.{LinkEnd}"
+  },
+  "welcomePageManageShortcutsChromium": {
+    "message": "Find shortcuts for this extension by going to {LinkStart}extension settings.{LinkEnd}"
+  },
+  "welcomePageTryFirefox": {
+    "message": "Want to Launch in Firefox? {LinkStart}Try our Chromium extension.{LinkEnd}"
+  },
+  "welcomePageTryChromium": {
+    "message": "Launch another browser with our {LinkStart}extension in Firefox.{LinkEnd}"
+  },
+  "welcomePageDefaultBrowser": {
+    "message": "(Default Browser)"
+  },
+  "welcomePageBeta": {
+    "message": "BETA"
+  },
+  "welcomePageNoExternalBrowserErrorChromium": {
+    "message": "Looks like Firefox isn't installed on this device. {LinkStart}Download Now{LinkEnd}"
```

```suggestion
    "message": "Looks like Firefox isn’t installed on this device. {LinkStart}Download Now{LinkEnd}"
```

Same applies to the next string.

---

PR: https://github.com/mozilla/firefox-launch/pull/21